concordium-std-derive/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## Unreleased changes
 
 - Removed `derive(Serial)` and `derive(Deserial)` (moved to `concordium-contracts-common-derive`).
+- Add ability to `derive(Reject)` for enums *with fields*, where all fields
+  implement the `Serial` trait.
```

Comment:
Something seems wrong with t his changelog entry diff. Can you make sure to merge with `main` before merging into main so we don't get a mangled changelog.

Reply:
Ah, I was too quick when merging in the main. Nice catch ;)